alacritty_terminal/src/term/mod.rs

```diff
@@ -938,6 +938,11 @@ impl Term {
         self.dynamic_title = config.dynamic_title();
         self.auto_scroll = config.scrolling.auto_scroll;
         self.grid.update_history(config.scrolling.history() as usize, &self.cursor.template);
+
+        if self.original_font_size == self.font_size {
+            self.original_font_size = config.font.size;
```

Comment:
I found one issue with the functionality of this PR. I think the original font size should be changed regardless of the current font size. So whenever resetting to the original, it would reset to the one set in the config.

Reply:
Oh, good catch I'll address that shortly.